doc/ranger.pod

```diff
@@ -302,6 +302,20 @@ The macro %space expands to a space character. You can use it to add spaces to
 the end of a command when needed, while preventing editors to strip spaces off
 the end of the line automatically.
 
+The macros %any and %any_path can be used when binding keys to
+commands. The macro %any will exapnd to the key pressed in the place of
+<any>. The macro %any_path will exapnd to the path of the bookmark which
+is mapped to the key that is pressed in the place of <any>. For example:
+
+	map c<any> echo %any
```

Comment:
Examples are a great idea but could you use more useful examples? These do show how it works but not why you might want to use them.

Reply:
Yes. There are actually a few examples in the default configuration file where %any and %any_path are used so I've used those to illustrate them.